content/src/policies/__id/delete.rego

```diff
@@ -1,25 +1,38 @@
 package peoplefinder.DELETE.api.users.__id
 
-import input.policy.path
+# Policy for deleting an employee in PeopleFinder
+#
+# Note: This policy is for illustration. The Delete button visibility is used by PeopleFinder, but the button does not call the delete API. 
```

Comment:
I think this is an app issue, not a policy issue, so it should be called out in the app. E.g. we can modify the app to actually call the delete API, and get back a message "This is a sample, so no user was actually harmed :)"

Reply:
Fixed